src/Components/Politicians/PoliticianCard/styles.js

```diff
@@ -0,0 +1,17 @@
+export const container = {
+  flex: '0 1 20%',
+  margin: '1%',
+  display: 'flex',
+  flexWrap: 'wrap',
+  alignItems: 'center',
+  textDecoration: 'none',
+};
+
+export const flexItem = {
+  flex: '1 1 100%',
+};
+
+export const name = {
```

Comment:
Hmm, first time I've seen an object spread in a style item.

Could you explain the reasoning behind it?
Rather than assigning two classes to the HTML element?